### DISCUSSION TITLE
Solve the inverse problem of stochastic PDEs containing the Ornstein–Uhlenbeck process (OU) process

### DISCUSSION THREAD

**Discussion by Ayi2222** [lululxvi/deepxde#1353] [Q&A]
Hello, Lu:

Recently I encounter an problem that can be modelled as an inverse problem of stochastic PDEs containing the OU process. The problem is as follow:

$$ Mdf(t)/dt + Df(t) = P(t) $$ 

where $P(t)$ is an O-U process, $M$ and $D$ are the coefficients to be solved. I tried to solve this proble with NN-arbitrary polynomial chaos (NN-aPC) based on DeepXDE. But I found that the PCE approximation of the OU process is very inaccurate, does this mean that NN-aPC cannot be used to solve this problem? Or for SDEs containing the OU process, do you have any recommendations for other solutions?

**Comment by lululxvi** [comment]
DeepXDE doesn't directly support SPDE. Do you implement the code by yourself?

I haven't solved SPDE with OU process, so I am not sure.
